6-anthos-install/README.md

```diff
@@ -1,5 +1,7 @@
 # Deploying Bank of Anthos
-These instructions need to be run on the bastion host. To access the bastion host, open the Google Cloud console, and navigate to Compute Engine in the GKE project. In my instance, `prj-bu1-d-boa-gke` Then, select the `gce-bastion-us-west1-b-01` and click on `ssh`. You need to be a whitelisted member to access this node.
+Before you begin, you need to be able to access the bastion host using IAP TCP forwarding. Follow these [instructions](https://cloud.google.com/iap/docs/using-tcp-forwarding#grant-permission) to grant this access to your user.
```

Comment:
We should expose `bastion_members` from here https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/blob/ae238f3cce0f9541e683ec14bb2f23d307e51354/5-infrastructure/modules/base-env/variables.tf#L73 in each env.

Reply:
@bharathkkb addressed in #50 this PR can be closed